src/context/authContext.js

```diff
@@ -0,0 +1,38 @@
+import { createContext, useContext, useEffect, useReducer } from "react";
+import { authReducer } from "../reducers/authReducer";
+
+const AuthContext=createContext({});
+const AuthProvider=({children})=>{
+const getoken=localStorage.getItem('sessiontoken')
+    const authInitialState={
+        user:[],
```

Comment:
Why are you using array here? Correct me if I am wrong, I think it should be an object (if you are planning to store the user after login).

Reply:
It was trying something forgot to remove this .removing it